File: Frontend/src/types/Partenaire.ts
export class Partenaire {
  id_partenaire: number;
  nom: string;
  type_activite: string;
  contact: string;
  adresse: string;

  constructor(
    id_partenaire: number,
    nom: string,
    type_activite: string,
    contact: string,
    adresse: string
  ) {
    this.id_partenaire = id_partenaire;
    this.nom = nom;
    this.type_activite = type_activite;
    this.contact = contact;
    this.adresse = adresse;
  }
}

File: Backend/src/main/java/com/example/demo/model/Partenaire.java
package com.example.demo.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "Partenaire")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Partenaire {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id_partenaire;

    private String nom;
    private String type_activite;
    private String contact;
    private String adresse;
}
